ouroboros-network/src/Ouroboros/Network/PeerSelection/LedgerPeers.hs

```diff
@@ -421,8 +423,8 @@ withLedgerPeers :: forall peerAddr resolver exception m a.
                      -> Async m Void
                      -> m a )
                 -> m a
-withLedgerPeers PeerActionsDNS { paToPeerAddr = toPeerAddr, paDnsActions = dnsActions, paDnsSemaphore = dnsSemaphore }
-                WithLedgerPeersArgs { wlpRng = inRng, wlpConsensusInterface = interface, wlpTracer = tracer, wlpGetUseLedgerPeers = readUseLedgerPeers }
+withLedgerPeers paDNS
+                wlpa
```

Comment:
We try to use long readable names rather than C stl nms :wink: I'd suggest `peerActionsDNS` and `ledgerPeersArgs`.  Another thing worth keeping in mind is consistency between calls.